templates/pdf_rag/utils.py

```diff
@@ -0,0 +1,249 @@
+import os
+import shutil
+from logging import getLogger
+from superduper import Model, logging
+
+
+def rematch(texts, answer, n=5):
+    texts_words = [set(t.lower().split()) for t in texts]
+    answer_words = set(answer.lower().split())
+    scores = [len(t & answer_words) / len(answer_words) for t in texts_words]
```

Comment:
may be use text to text distance metric?

Reply:
The calculation method can be changed, so let’s ignore it for now. A simple implementation was done here—chunks with more words that match the answer are displayed earlier.